.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
```

Comment:
Does this file need to be here?

Reply:
Yea, it'll avoid DS_Store files accidentally being committed to master by other people using macOS.

Reply:
Perfect, love it.